.gitignore

```diff
@@ -0,0 +1,11 @@
+/Dungeon.class
```

Comment:
I resolved some conflicting things with master and your branch and the .gitignore file here so it might look a little different. Lets add .project and .classpath here